curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md

```diff
@@ -186,25 +188,25 @@ The `.moon` element should be adjusted horizontally by translating it to `-50%`
 assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.moon')?.transform, 'translateX(-50%)');
 ```
 
-Your `.earth` element should have a background color.
+The `.earth` selector should have a background color.
 
 ```js
 assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('.earth')?.backgroundColor);
 ```
 
-Your `.earth` element should have `border-radius` of `50%` to make it look like a circle.
+The `.earth` selector should have `border-radius` of `50%` to make it look like a circle.
```

Comment:
```suggestion
The `.earth` selector should have `border-radius` of `50%`.
```

this is not specified anymore in the user story so we can remove it from here too